core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.html

```diff
@@ -73,6 +73,7 @@
                     >
                         <dotcms-template-builder-box
                             class="grid-stack-item-content"
+                            [containerMap]="containerMap$ | async"
```

Comment:
```suggestion
                            [containerMap]="vm .containerMap"
```

I think this is the same.